### ISSUE TITLE
newly crashing vst: Majken's Chimera

### ISSUE THREAD

**Issue by jmckernon** [robbert-vdh/yabridge#199]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

Since a system upgrade, a certain plugin, Majken's Chimera, now crashes when I try to scan it in Reaper. It was working before my upgrade (i.e. in previous versions of wine and yabridge, I believe 3.8.1 of yabridge, not sure of which wine).

I tried downgrading to confirm which versions it worked with, but I couldn't get the downgrade to work properly. On the other hand, I did get the plugin to work from a reaper installation within wine, so I think the issue is not with wine itself.

The plugin is free and available here: http://www.majken.se/

Have run `yabridgectl sync`.

### What did you expect to happen?

Plugin to scan.

### What actually happened?

Plugin did not scan, wine threw some kind of an error, looks like possible a segfault, not sure.

### Operating system

Arch

### How did you install yabridge?

yay/pacman

### yabridge version

4.0.2

### yabridgectl version

4.0.2

### Wine version

7.16

### Plugin

Majken's Chimera

### Plugin type

VST2

### Plugin architecture

32-bit

### Host

Reaper 6.66

### Debug log

```18:19:28 [chimera-05MNZgnx] Initializing yabridge version 4.0.2
18:19:28 [chimera-05MNZgnx] library:       '/usr/lib/libyabridge-vst2.so'
18:19:28 [chimera-05MNZgnx] host:          '/usr/bin/yabridge-host-32.exe'
18:19:28 [chimera-05MNZgnx] plugin:        '/home/james/vstwine/drive_c/vst32/chimera.dll'
18:19:28 [chimera-05MNZgnx] plugin type:   'VST2'
18:19:28 [chimera-05MNZgnx] realtime:      'yes'
18:19:28 [chimera-05MNZgnx] sockets:       '/run/user/1000/yabridge-chimera-05MNZgnx'
18:19:28 [chimera-05MNZgnx] wine prefix:   '/home/james/vstwine'
18:19:28 [chimera-05MNZgnx] wine version:  '7.16'
18:19:28 [chimera-05MNZgnx] 
18:19:28 [chimera-05MNZgnx] config from:   '<defaults>'
18:19:28 [chimera-05MNZgnx] hosting mode:  'individually, 32-bit'
18:19:28 [chimera-05MNZgnx] other options: '<none>'
18:19:28 [chimera-05MNZgnx] 
18:19:28 [chimera-05MNZgnx] Enabled features:
18:19:28 [chimera-05MNZgnx] - bitbridge support
18:19:28 [chimera-05MNZgnx] - VST3 support
18:19:28 [chimera-05MNZgnx] 
18:19:28 [chimera-05MNZgnx] [Wine STDERR] 0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
18:19:28 [chimera-05MNZgnx] [Wine STDERR] 0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
18:19:28 [chimera-05MNZgnx] [Wine STDERR] 0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
18:19:28 [chimera-05MNZgnx] [Wine STDERR] 0084:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
18:19:28 [chimera-05MNZgnx] [Wine STDERR] Initializing yabridge host version 4.0.2 (32-bit compatibility mode)
18:19:28 [chimera-05MNZgnx] [Wine STDERR] Preparing to load VST2 plugin at '/home/james/vstwine/drive_c/vst32/chimera.dll'
18:19:28 [chimera-05MNZgnx] >> audioMasterCallback() audioMasterVersion(index = 0, value = 0, option = 0, data = nullptr)
18:19:28 [chimera-05MNZgnx]    audioMasterCallback() :: 2400
18:19:28 [chimera-05MNZgnx] [Wine STDERR] 0024:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
18:19:29 [chimera-05MNZgnx] >> audioMasterCallback() audioMasterUpdateDisplay(index = 0, value = 0, option = 0, data = nullptr)
18:19:29 [chimera-05MNZgnx]    audioMasterCallback() :: 1
18:19:29 [chimera-05MNZgnx] [Wine STDERR] wine: Unhandled page fault on read access to F7800438 at address F7800438 (thread 0100), starting debugger...
18:19:29 [chimera-05MNZgnx] [Wine STDERR] 0074:fixme:imm:ImeSetActiveContext (0000000000010026, 0): stub
18:19:29 [chimera-05MNZgnx] [Wine STDERR] 0074:fixme:imm:ImmReleaseContext (0000000000010020, 0000000000010026): stub
18:19:29 [chimera-05MNZgnx] [Wine STDERR] 0110:fixme:imm:ImeSetActiveContext (0000000000010066, 1): stub
18:19:29 [chimera-05MNZgnx] [Wine STDERR] 0110:fixme:imm:ImmReleaseContext (000000000002005E, 0000000000010066): stub
18:19:29 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:imm:ImeSetActiveContext (0001008A, 1): stub
18:19:29 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:imm:ImmReleaseContext (00010094, 0001008A): stub
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:31 [chimera-05MNZgnx] [Wine STDERR] 0108:fixme:dbghelp:elf_search_auxv can't find symbol in module
18:19:36 [error] 
18:19:36 [error] Error during initialization:
18:19:36 [error] read: End of file
18:19:36 [error]
```

### Anything else?

Let me know whether you're able to reproduce. Thanks.

**Comment by jmckernon** [comment]
I'll close this for now, since I haven't had a chance to look much into it since I initially opened the issue (and to keep your project neat and tidy). I did try some of what you suggested, and couldn't get it working, but now I can't remember what I tried. I'll reopen if I do manage to take some time troubleshooting it and it seems to be an issue with yabridge.

I hope you don't mind my opening issues on these obscure old VSTs! (The last one I opened, for a memory leak in PSP N2O, seems to be resolved for me now, presumably just through an update to wine.)
